producer

```diff
@@ -0,0 +1,22 @@
+from entities import *
+from config import Config
+from random import random
+from random import randint
+
+
+class Producer(Seller):
+    def __init__(self):
+        super(Producer, self).__init__()
+
+    def produce(self):
+        for _ in range(*Config.production_rate):
+            destination_shipment= [randint(*Config.region[0]),  # x axis
+                            randint(*Config.region[1])]  # y axis
+            shipments.append([self.location, destination_shipment, id(seller)])
```

Comment:
This should _not_ be a List, but rather a new Object (i.e. define a class `Shipment` with the appropriate member fields).

Reply:
Then you could do,

```python
shipments.append(new Shipment(self.location, destination, seller.id))
```

`id(seller)` does not do what you may think it does.